libraries/freertos_plus/standard/freertos_plus_posix/source/FreeRTOS_POSIX_pthread_cond.c

```diff
@@ -77,6 +77,25 @@ static void prvInitializeStaticCond( pthread_cond_internal_t * pxCond )
         taskEXIT_CRITICAL();
     }
 }
+/* @brief Check "atomically" if iLocalWaitingThreads == pxCond->iWaitingThreads and decrement.  
```

Comment:
I believe it has to start with /** (two stars) for Doxygen to pick up the brief?

Reply:
Fixed!